### ISSUE TITLE
Amazon GameLift resource tagging

### ISSUE THREAD

**Issue by ewbankkit** [terraform-providers/terraform-provider-aws#11384]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Amazon GameLift now supports resource tagging.

### New or Affected Resource(s)

* [`aws_gamelift_alias`](https://www.terraform.io/docs/providers/aws/r/gamelift_alias.html)
* [`aws_gamelift_build`](https://www.terraform.io/docs/providers/aws/r/gamelift_build.html)
* [`aws_gamelift_fleet`](https://www.terraform.io/docs/providers/aws/r/gamelift_fleet.html)
* [`aws_gamelift_game_session_queue`](https://www.terraform.io/docs/providers/aws/r/gamelift_game_session_queue.html)

### Potential Terraform Configuration

```hcl
resource "aws_gamelift_alias" "example" {
  tags = {
    Usage= "For example use only"
  }
}
```

### References

[Announcement](https://aws.amazon.com/about-aws/whats-new/2019/12/new-consistent-authorization-experience-for-amazon-gamelift/).
[API reference](https://docs.aws.amazon.com/gamelift/latest/apireference/API_TagResource.html).

Requires [AWS SDK v1.26.6](https://github.com/aws/aws-sdk-go/releases/tag/v1.26.6):
* https://github.com/terraform-providers/terraform-provider-aws/pull/11375

**Comment by bflad** [comment]
Support for tagging `aws_gamelift_alias`, `aws_gamelift_build`, and `aws_gamelift_game_session_queue` resources has been merged and will release with version 2.45.0 of the Terraform AWS Provider, Thursday next week. Thanks to @DrFaust92 for the implementation. 👍

**Comment by bflad** [comment]
Support for tagging `aws_gamelift_fleet` has been merged and will release with version 2.49.0 of the Terraform AWS Provider, Thursday this week. Thanks to @DrFaust92 for the implementation. 👍
